### PR TITLE
fix: add new award and sorting

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[維運] 關於我們資料更新、增加區塊](https://app.asana.com/0/1203699264568808/1205329976610313/f) defects

# Notice

# Dependency

### CODE REVIEW

File: src/components/about-us/constants/section-02/category-ids.js [hunk 1]
```diff
@@ -8,8 +8,8 @@ const categoryIds = {
   engineer: 'engineer',
   designer: 'designer',
   podcast: 'podcast',
-  theReporterForKids: 'theReporterForKids',
   dataJournalist: 'dataJournalist',
+  theReporterForKids: 'theReporterForKids',
```
Comment (Aylie-Chou) [inline src/components/about-us/constants/section-02/category-ids.js:12]: Question: if key orders matters in `categoryIds`, could we modify its type to array or array of object ?

File: src/components/about-us/constants/section-03/awards-name.json [hunk 2]
```diff
@@ -49,6 +49,11 @@
     "award": "KKBOX Podcast 風雲榜",
     "awardEng": "KKBOX Podcast"
   },
+  {
+    "awardId": "SND",
+    "award": "SND最佳新聞設計創意競賽",
```
Comment (Aylie-Chou) [inline src/components/about-us/constants/section-03/awards-name.json:54]: 盤古之白？

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Please have a look at comments ~

File: src/components/about-us/constants/section-02/category-ids.js (hunk 1 above)
Comment (liruchen32) [reply src/components/about-us/constants/section-02/category-ids.js:12]: fixed

File: src/components/about-us/constants/section-03/awards-name.json (hunk 2 above)
Comment (liruchen32) [reply src/components/about-us/constants/section-03/awards-name.json]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**